include/openssl/pkcs12.h.in

```diff
@@ -45,6 +45,7 @@ extern "C" {
 
 # define PKCS12_MAC_KEY_LENGTH 20
 
+/* Unused macro kept for backwards compatibility */
```

Comment:
The macro is still used in `crypto/pkcs12/p12_mutl.c`, I thought the idea was to change that use to a 16 and dropping the macro.

Reply:
Updated the ccomment, As this was in the exposed headfile, and kept for the backword complalibility consider.